src/platform/graphics/constants.js

```diff
@@ -946,21 +946,109 @@ export const PIXELFORMAT_R8 = 52;
+ *
+ * @type {number}
+ * @category Graphics
+ */
+export const PIXELFORMAT_DXT3_SRGB = 55;
+
+/**
+ * Format equivalent to {@link PIXELFORMAT_DXT5} but sampled in linear color space.
+ *
+ * @type {number}
+ * @category Graphics
+ */
+export const PIXELFORMAT_DXT5_SRGB = 56;
+
+/**
+ * Format equivalent to {@link PIXELFORMAT_PVRTC_2BPP_RGB_1} but sampled in linear color space.
+ *
+ * @type {number}
+ * @category Graphics
+ */
+export const PIXELFORMAT_PVRTC_2BPP_SRGB_1 = 57;
+
+/**
+ * Format equivalent to {@link PIXELFORMAT_PVRTC_2BPP_RGBA_1} but sampled in linear color space.
+ *
+ * @type {number}
+ * @category Graphics
+ */
+export const PIXELFORMAT_PVRTC_2BPP_SRGBA_1 = 58;
+
+/**
+ * Format equivalent to {@link PIXELFORMAT_PVRTC_4BPP_RGB_1} but sampled in linear color space.
+ *
+ * @type {number}
+ * @category Graphics
+ */
+export const PIXELFORMAT_PVRTC_4BPP_SRGB_1 = 59;
+
+/**
+ * Format equivalent to {@link PIXELFORMAT_PVRTC_4BPP_RGBA_1} but sampled in linear color space.
+ *
+ * @type {number}
+ * @category Graphics
+ */
+export const PIXELFORMAT_PVRTC_4BPP_SRGBA_1 = 60;
+
+/**
+ * Format equivalent to {@link PIXELFORMAT_ETC2_RGB} but sampled in linear color space.
+ *
+ * @type {number}
+ * @category Graphics
+ */
+export const PIXELFORMAT_ETC2_SRGB = 61;
+
+/**
+ * Format equivalent to {@link PIXELFORMAT_ETC2_RGBA} but sampled in linear color space.
+ *
+ * @type {number}
+ * @category Graphics
+ */
+export const PIXELFORMAT_ETC2_SRGBA = 62;
+
+/**
+ * Format equivalent to {@link PIXELFORMAT_ASTC_4x4} but sampled in linear color space.
+ *
+ * @type {number}
+ * @category Graphics
+ */
+export const PIXELFORMAT_ASTC_4x4_SRGB = 63;
+
+/**
+ * Information about pixel formats.
+ *
+ * ldr: whether the format is low dynamic range (LDR), which typically means it's not HDR, and uses
+ * sRGB color space to store the color values
+ * srgbFormat: the corresponding sRGB format (which automatically converts the sRGB value to linear)
+ *
+ * @type {Map<number, { name: string, size?: number, blockSize?: number, ldr?: boolean, srgb?: boolean, srgbFormat?: number, isInt?: boolean }>}
+ * @ignore
+ */
 export const pixelFormatInfo = new Map([
 
     // float formats
-    [PIXELFORMAT_A8,            { name: 'A8', size: 1 }],
-    [PIXELFORMAT_R8,            { name: 'R8', size: 1, srgb: true }],
-    [PIXELFORMAT_L8,            { name: 'L8', size: 1 }],
-    [PIXELFORMAT_LA8,           { name: 'LA8', size: 2 }],
-    [PIXELFORMAT_RG8,           { name: 'RG8', size: 2, srgb: true }],
-    [PIXELFORMAT_RGB565,        { name: 'RGB565', size: 2, srgb: true }],
-    [PIXELFORMAT_RGBA5551,      { name: 'RGBA5551', size: 2, srgb: true }],
-    [PIXELFORMAT_RGBA4,         { name: 'RGBA4', size: 2, srgb: true }],
-    [PIXELFORMAT_RGB8,          { name: 'RGB8', size: 4, srgb: true }],
-    [PIXELFORMAT_RGBA8,         { name: 'RGBA8', size: 4, srgb: true }],
+    [PIXELFORMAT_A8,            { name: 'A8', size: 1, ldr: true }],
```

Comment:
I still think it would be neater to flag the few hdr formats as `hdr` rather than the relatively many formats as `ldr`. (Integer formats aren't color formats and so shouldn't get either flag).